packages/database/src/api/Database.ts

```diff
@@ -320,9 +320,11 @@ export function getDatabase(
   const db = _getProvider(app, 'database').getImmediate({
     identifier: url
   }) as Database;
-  const emulator = getDefaultEmulatorHostnameAndPort('database');
-  if (emulator) {
-    connectDatabaseEmulator(db, ...emulator);
+  if (!db._instanceStarted) {
+    const emulator = getDefaultEmulatorHostnameAndPort('database');
+    if (emulator) {
+      connectDatabaseEmulator(db, ...emulator);
+    }
   }
   return db;
 }
```

Comment:
So, per the issue, can we remove the `if (db._instanceStarted)` on L353 in `connectDatabaseEmulator` now?

Reply:
`connectDatabaseEmulator` is still a public function, so the user could still call it outside of the normal flow